lib/std/json.zig

```diff
@@ -3257,3 +3257,23 @@ test "stringify struct with custom stringifier" {
 test "stringify vector" {
     try teststringify("[1,1]", @splat(2, @as(u32, 1)), StringifyOptions{});
 }
+
+// Same as `stringify` but accepts an Allocator and stores result in dynamically allocated memory instead of using a Writer.
+// Caller owns returned memory.
+pub fn stringifyAlloc(allocator: *std.mem.Allocator, options: StringifyOptions, value: anytype) ![]const u8 {
```

Comment:
to match `stringify`, should `options` come after `value`?

Reply:
I switched the order because if someone is passing the values inline it is significantly more likely that the `value` parameter is going to be spread amongst multiple lines.

Reply:
fixed